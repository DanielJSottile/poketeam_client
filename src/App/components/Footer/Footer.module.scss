@import '../../../index.scss';

.small-logo-image {
  width: 25%;
}

.no-styling {
  font-size: 12px;
  background-color: transparent;
  color: $super-light-cream;
  text-decoration: underline;
  &:hover {
    background-color: transparent;
    color: $azure;
  }
}
