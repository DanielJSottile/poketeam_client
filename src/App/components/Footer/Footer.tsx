import React, { FunctionComponent } from 'react';
import Image from '../Image';
import PokeTeamsLogo from '../../Images/PokeTeams.png';
import styles from './Footer.module.scss';

const Footer: FunctionComponent = () => {
  return (
    <footer>
      <Image
        imageClass={styles['small-logo-image']}
        src={PokeTeamsLogo}
        alt="PokeTeams Logo"
      />
      <span>{`Created by Daniel Sottile || © July 2021 || Pokémon is © 1995-2021 Nintendo`}</span>
    </footer>
  );
};

export default Footer;
