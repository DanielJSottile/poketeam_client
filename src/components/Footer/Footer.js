import React, { Component } from 'react'

export default class Footer extends Component {
  render() {
    return (
      <footer>{`Created by Daniel Sottile || &copy 2020`}</footer>
    );
  };
};
