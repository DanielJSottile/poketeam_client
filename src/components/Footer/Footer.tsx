import React from 'react';
import './Footer.scss';

// Component

const Footer = (props: any) => {
  // Final Render

  return (
    <footer>{`Created by Daniel Sottile || © May 2020 || Pokémon is © 1995-2020 Nintendo`}</footer>
  );
};

export default Footer;
