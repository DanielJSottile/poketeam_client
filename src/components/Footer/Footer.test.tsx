import React from "react";
import ReactDOM from "react-dom";
import { BrowserRouter } from "react-router-dom";
//import { render } from '@testing-library/react';
import Footer from "./Footer";
import { shallow, mount, render } from "enzyme";
import toJson from "enzyme-to-json";

describe("<LoginForm/>", (): void => {
  it("renders without crashing", (): void => {
    const div: HTMLDivElement = document.createElement("div");
    ReactDOM.render(
      <BrowserRouter>
        <Footer />
      </BrowserRouter>,
      div
    );
    ReactDOM.unmountComponentAtNode(div);
  });
  it("snapshot", (): void => {
    const wrapper = shallow(<Footer />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
