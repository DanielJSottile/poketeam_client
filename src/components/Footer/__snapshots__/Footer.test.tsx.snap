// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LoginForm/> snapshot 1`] = `
<footer>
  Created by Daniel Sottile || © May 2020 || Pokémon is © 1995-2020 Nintendo
</footer>
`;
